src/ext.js

```diff
@@ -65,6 +65,14 @@ export default function ext(/* env */) {
               defaultValue: false,
               show: data => checkShow(data, 'softLock'),
             },
+            value: {
+              // TODO: expressions
+              type: 'string',
+              ref: 'value',
+              component: 'string',
+              label: 'Value',
+              show: data => checkShow(data, 'value'),
+            },
```

Comment:
Should we create a new user story for expressions? I know Tobias was looking into this as well